silhouette/Graphtec.py

```diff
@@ -395,7 +395,7 @@ def __init__(self, log=sys.stderr):
       print("Searching for bt devices on OSX", file=log)
       # on osx a rfcomm port is created and linked automatically when the cameo is connected
       # TODO add support for other machines. I would expect files like CAMEO3-... and PORTRAIT2-...
-      pat = re.compile("tty\.(PORTRAIT2|CAMEO3|CAMEO4)-.*")
```

Comment:
Are you sure about this? Normally I would expect the old code to match a dot character (".") whereas the new code would match any character but newlines?